docs/pages/versions/unversioned/workflow/configuration.md

```diff
@@ -431,6 +431,14 @@ Configuration for how and when the app should request OTA JavaScript updates
     */
     "usesAppleSignIn": BOOLEAN,
 
+    /*
+      A boolean value indicating if the app may access the notes stored in contacts.
+      See Contacts docs for details.
+
+      ExpoKit: use Xcode to set this.
+    */
+    "note": BOOLEAN,
```

Comment:
name of this configuration should be self-descriptive, `note` tells nothing to the users